### PR TITLE
Set retry flag true on Prep Upload

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: lib/src/main/java/com/smileidentity/viewmodel/document/OrchestratedDocumentViewModel.kt [hunk 1]
```diff
@@ -184,7 +184,17 @@ internal abstract class OrchestratedDocumentViewModel<T : Parcelable>(
                 timestamp = authResponse.timestamp,
             )
 
-            val prepUploadResponse = SmileID.api.prepUpload(prepUploadRequest)
+            val prepUploadResponse = try {
+                SmileID.api.prepUpload(prepUploadRequest)
+            } catch (e: SmileIDException) {
+                // It may be the case that Prep Upload was called during the job but the link expired.
+                // We need to pass retry=true in order to obtain a new link
+                if (e.details.code == "2215") {
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply lib/src/main/java/com/smileidentity/viewmodel/document/OrchestratedDocumentViewModel.kt:192]: Explained on Slack
